source/common/tracing/http_tracer_impl.cc

```diff
@@ -87,42 +87,36 @@ template <class T> static void addGrpcResponseTags(Span& span, const T& headers)
   // Set error tag when Grpc status code represents an upstream error. See
   // https://github.com/envoyproxy/envoy/issues/18877.
   absl::optional<Grpc::Status::GrpcStatus> grpc_status_code = Grpc::Common::getGrpcStatus(headers);
-  if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.update_grpc_response_error_tag")) {
```

Comment:
The old logic is that
```
if feature
  if
else
```

Now we have enabled the feature, still need the `else` on the top level?

Reply:
I think we don't need `else` any more, because the feature is always enabled.

Reply:
True. The logic in `else` is actually the old code path which we don't need anymore.